get_user_input.py

```diff
@@ -1,82 +1,119 @@
+    parser.add_argument("-dt", "--time_steps", type=float, default=100,
+                        required=False,
                         help="The number of time nodes in the solution.")
 
-    parser.add_argument("-y", "--yield_strength", type=float, default=250, required=False,
-                        help="Yield strength of string.")
+    parser.add_argument("-y", "--yield_stress", type=float, default=215,
+                        required=False,
+                        help="Yield strength of string material.")
+
+    parser.add_argument("-cn", "--courant_number", type=float, default=0.75,
+                        required=False, help="The Courant Number is " +
+                        "u*dt*dx and must be less than 1.")
 
-    parser.add_argument("-cn", "--courant_number", type=float, default=0.75, required=False,
-                        help="The Courant Number is u*dt*dx and must be less than 1.")
+    parser.add_argument("-v0", "--initial_velocity", type=float, default=0,
+                        required=False, help="Initial velocity of the " +
+                        "string at the pluck position")
 
-    parser.add_argument("-v0", "--initial_velocity", type=float, default=0, required=False,
-                        help="Initial velocity of the string at the pluck position") 
+    parser.add_argument("-c", "--wave_speed", type=float, default=330,
+                        required=False, help="Speed of the wave in meters" +
+                        " per second.")
 
-    parser.add_argument("-c", "--wave_speed", type=float, default=330, required=False,
-                        help="Speed of the wave in meters per second.")
+    parser.add_argument("-E", "--youngs_modulus", type=float, default=190000,
+                        required=False, help="Young's modulus of material.")
 
     args = parser.parse_args()
 
     check_pluck_position(args.string_length, args.pluck_position)
-    check_string_length(args.string_length, args.pluck_position, args.pluck_displacement, args.yield_strength)
+    check_pluck_displacement(args.string_length, args.pluck_position,
+                             args.pluck_displacement, args.yield_stress,
+                             args.youngs_modulus)
     check_courant_number(args.courant_number)
+    check_plotting_times(args.time, args.plot_times)
 
-    user_input = {'String Length' : args.string_length, 
-                  'Pluck Position' : args.pluck_position,
-                  'Pluck Displacement' : args.pluck_displacement,
-                  'Time' : args.time, 
-                  'Time Steps' : args.time_steps,
-                  'Plotting Times' : args.plot_times,
-                  'Yield Strength' : args.yield_strength,
-                  'Courant Number' : args.courant_number,
-                  'Initial Velocity' : args.initial_velocity,
-                  'Wave Speed' : args.wave_speed}
+    user_input = {'String Length': args.string_length,
```

Comment:
Does switching to this dictionary make it easier to use/read?  Why not just pass the `args` object around?

Reply:
(This is a data model change that everyone should discuss/agree to...)